Introduction.md

```diff
@@ -2,60 +2,59 @@
 
 ## Maturity levels
 
-Corporate adoption of open source can typically be described using a model of maturity levels. These levels characterize how, with growing maturity, open source is used in an increasingly effective fashion to drive value and realize business needs. How outbound open source is handled is a distinguishing factor between the different levels of maturity. A critical factor is the insight that influencing the open source ecosystem is mainly done by participation and contributing to open source projects.
+Corporate adoption of open source software (OSS) can typically be classified with a model of maturity levels. These levels describe how OSS is used in an increasingly effective fashion to drive value and address business needs. One of the distinguishing factors for the different maturity levels is how outbound open source is handled in an organization. The insight that influencing the open source ecosystem is mainly done by participation in and contributing to open source projects is seen as a critcal factor.
 
-A typical maturity model of corporate open source adoption will look like this (see for example [Haddad: Open Source Program Offices](https://www.linkedin.com/pulse/open-source-program-offices-primer-organizational-roles-haddad)):
+A typical maturity model of corporate open source adoption looks like this (see for example [Haddad: Open Source Program Offices](https://www.linkedin.com/pulse/open-source-program-offices-primer-organizational-roles-haddad)):
 
 0. Denial - No or unconscious use of open source
 1. Consumption - Passive use of open source software
-2. Participation - Engagement with open source community
+2. Participation - Engagement with open source communities
 3. Contribution - Pragmatic contributions to open source projects
 4. Leadership - Strategic involvement with open source to drive business value
 
-To advance from one level to another it requires certain initiatives and structural and organizational elements.
+To advance from one level to another, certain initiatives and structural and organizational elements are required.
 
-Going from consumption to participation will start with informal engagement and low-effort activities such as reporting bugs in upstream projects driven by technical needs. Decisions about open source contributions will be ad-hoc and be taken for individual cases.
+Going from consumption to participation, for example, will start with informal engagement and low-effort activities such as reporting bugs in upstream projects, which typically is driven by technical needs. On thet level, decisions about open source contributions will normally be ad-hoc and be taken for individual cases only.
```

Comment:
Small typo:

```suggestion
Going from consumption to participation, for example, will start with informal engagement and low-effort activities such as reporting bugs in upstream projects, which typically is driven by technical needs. On that level, decisions about open source contributions will normally be ad-hoc and be taken for individual cases only.
```

I'm also not sure, if the "for example" is the right term here. We say for each level how to advance to the next one, so it's not really an example, but one part of the complete list.